Source/Bloodmasters.sln

```diff
@@ -17,42 +17,40 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{FAC8
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{1FA71E58-528D-4FF0-A00E-E535C0815317}"
 	ProjectSection(SolutionItems) = preProject
-		..\.github\workflows\main.yml = ..\.github\workflows\main.yml
 		Directory.Build.props = Directory.Build.props
+		..\.github\workflows\main.yml = ..\.github\workflows\main.yml
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Bloodmasters.Shared", "Shared\Bloodmasters.Shared.csproj", "{392D7C6F-B977-4507-A10E-12AD6F02A617}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Bloodmasters.Shared", "Shared\Bloodmasters.Shared.csproj", "{392D7C6F-B977-4507-A10E-12AD6F02A617}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|x86 = Debug|x86
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Debug|x86.ActiveCfg = Debug|x86
-		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Debug|x86.Build.0 = Debug|x86
+		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Debug|x86.Build.0 = Debug|Any CPU
 		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Release|x86.ActiveCfg = Release|x86
 		{406F2DC0-ED64-4D82-A75C-12DFC3300A66}.Release|x86.Build.0 = Release|x86
 		{0232A66E-1D9B-483C-88A3-432F4EE78D12}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{0232A66E-1D9B-483C-88A3-432F4EE78D12}.Debug|x86.Build.0 = Debug|Any CPU
 		{0232A66E-1D9B-483C-88A3-432F4EE78D12}.Release|x86.ActiveCfg = Release|Any CPU
 		{0232A66E-1D9B-483C-88A3-432F4EE78D12}.Release|x86.Build.0 = Release|Any CPU
-		{45784815-6BB5-47B0-A4EA-78E3B1A4E098}.Debug|x86.ActiveCfg = Debug|x86
-		{45784815-6BB5-47B0-A4EA-78E3B1A4E098}.Debug|x86.Build.0 = Debug|x86
+		{45784815-6BB5-47B0-A4EA-78E3B1A4E098}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{45784815-6BB5-47B0-A4EA-78E3B1A4E098}.Release|x86.ActiveCfg = Release|x86
 		{45784815-6BB5-47B0-A4EA-78E3B1A4E098}.Release|x86.Build.0 = Release|x86
 		{392D7C6F-B977-4507-A10E-12AD6F02A617}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{392D7C6F-B977-4507-A10E-12AD6F02A617}.Debug|x86.Build.0 = Debug|Any CPU
 		{392D7C6F-B977-4507-A10E-12AD6F02A617}.Release|x86.ActiveCfg = Release|Any CPU
 		{392D7C6F-B977-4507-A10E-12AD6F02A617}.Release|x86.Build.0 = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
```

Comment:
Why change this configuration? Please avoid. I'll clean it up later to universally use AnyCPU, but for now it shouldn't be changed together with the audio lib. Or should it?